src/redux/reducers/book.js

```diff
@@ -0,0 +1,26 @@
+// import randomIdGenerator from '../../helpers/randomId';
```

Comment:
* [ ] Kindly remove commented code from your code to make it clean and reduce size.

Reply:
Done :heavy_check_mark: